components/calendar/src/date.rs

```diff
@@ -89,6 +89,8 @@ impl<C> Deref for Ref<'_, C> {
 
 /// A date for a given calendar.
 ///
+/// **This type originates from the [`icu_calendar`](docs.rs/icu_calendar) crate, though it may be found reexported elsewhere.**
```

Comment:
Suggestion, here and elsewhere:

```suggestion
/// **The primary definition of this type is in the [`icu_calendar`](docs.rs/icu_calendar) crate.
/// Other ICU4X crates re-export it for convenience.**
```